plugins/defaultplugins/common/model/nat/nat.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package nat;
+
+/* NAT44 global config */
+message Nat44Global {
+    uint32 vrfId = 1;                           /* VRF (table) ID */ 
+    bool forwarding = 2;                        /* Enable/disable forwarding */
+    repeated string in = 3;                     /* Local network interfaces enabled for NAT44 */
+    repeated string out = 4;                    /* External network interfaces enabled for NAT44 */
```

Comment:
For each interface we will also need to enable/disable "output feature" (nat44_interface_add_del_output_feature + nat44_interface_output_feature_dump).

Instead of string type for "in" & "out" we will probably need something like:

message Interface {
 string name;
 bool outputFeature;
}